airflow/providers/amazon/provider.yaml

```diff
@@ -76,7 +76,9 @@ dependencies:
   - sqlalchemy_redshift>=0.8.6
   - mypy-boto3-rds>=1.24.0
   - mypy-boto3-redshift-data>=1.24.0
-  - mypy-boto3-appflow>=1.24.0
+  # exclude 1.28.12 as it causes strange typing inconsistency
+  # https://github.com/youtype/mypy_boto3_builder/issues/209
+  - mypy-boto3-appflow>=1.24.0,!=1.28.12
```

Comment:
We're excluding just this single patch release right? Do you have confidence that it will be fixed by the next? Or you'd like to just wait and see?

Reply:
Chatted offline, it is the latter, a mechanism to see if it's still broken in future releases. I'd perhaps exclude all 1.28 patch releases and wait for the next minor, but we can always take that step later